.github/workflows/release_workflow.yml

```
@@ -0,0 +1,40 @@
+name: Build and publish
+
+on:
+  push:
+
+jobs:
+  build:
+    name: Build wheel and sdist
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        ref: ${{ github.ref }}
+    - uses: hynek/build-and-inspect-python-package@v2
+
+  publish:
+    name: Publish to PyPI
+    if: startsWith(github.ref, 'refs/tags/')
+    runs-on: ubuntu-latest
+    environment:
+      name: publishing
```

```suggestion
      name: pypi
```

https://github.com/MeltanoLabs/tap-mysql/issues/26